### PR TITLE
Modified exists api to support multiple param set related to auth.

### PR DESCRIPTION
**Pull request by ragupta-git**
Modified exists api to support multiple param set related to auth.

### CODE REVIEW

File: ucsm_apis/admin/auth.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def auth_domain_realm_configure(handle, domain_name, realm="local",
         domain_name (string): auth domain name
         realm (string): realm ["ldap", "local", "none", "radius", "tacacs"]
                         Use "none" to disable auth
-        use2_factor(bool): Two Factor Authentication (True/False)
+        use2_factor(bool): Two Factor Authentication ["yes", "no"]
```
Comment (vvb) [inline ucsm_apis/admin/auth.py]: type `bool` -> `str`

Comment (ragupta-git) [inline ucsm_apis/admin/auth.py]: Done.

File: ucsm_apis/admin/auth.py [hunk 2]
```diff
@@ -212,8 +248,11 @@ def native_auth_configure(handle, def_role_policy=None,
     Args:
         handle (UcsHandle)
         def_role_policy (string): def_role_policy
+         valid values are "assign-default-role", "no-login"
```
Comment (ragupta-git) [inline ucsm_apis/admin/auth.py]: better description def_role_policy

File: ucsm_apis/admin/auth.py [hunk 3]
```diff
@@ -244,19 +285,48 @@ def native_auth_configure(handle, def_role_policy=None,
...
 
     Args:
         handle (UcsHandle)
         realm (string): realm ["ldap", "local", "none", "radius", "tacacs"]
                         Use "none" to disable auth
-        session_timeout (string): session_timeout
         refresh_period (string): refresh_period
+        session_timeout (string): session_timeout
         provider_group (string): provider_group
+        use2_factor (str): two factor authentication ["yes", "no"]
```
Comment (ragupta-git) [inline ucsm_apis/admin/auth.py]: type str -> string

File: ucsm_apis/admin/auth.py [hunk 4]
```diff
@@ -289,16 +366,51 @@ def native_auth_default(handle, realm=None, session_timeout=None,
...
     """
     configure console native authentication.
 
     Args:
         handle (UcsHandle)
         realm (string): realm ["ldap", "local", "none", "radius", "tacacs"]
                         Use "none" to disable auth
-        provider_group (string): provider_group
+        provider_group (string): provider group name
+        use2_factor (str): two factor authentication ["yes", "no"]
```
Comment (ragupta-git) [inline ucsm_apis/admin/auth.py]: str -> string

**Comment by ragupta-git** [comment]
Incorporated review comments.

**Pull request merged**